### DISCUSSION TITLE
PLEASE Clean up your program

### DISCUSSION THREAD

**Discussion by Mikrizi** [rustdesk/rustdesk#2637] [General]
I beg you, make a separate version, or untie the main one from the mandatory use of DBUS, SystemD, pulseaudio

No way to use rustdesk on an init-freedom system, (like Hyperbola GNU/linux), thanks!

**Comment by roostercactus** [comment]
The appimage (nightly build) runs fine on Devuan.  I'm just doing a direct connect (not using Rustdesk server) over VPN between a Beowulf and Daedalus laptop and it connects.  I haven't tested anything other than just remote control so I don't know if audio/cut/paste/clipboard/file/sharing works.  But simple remote control does work.
